spec/rubocop/cop/lint/void_spec.rb

```diff
@@ -1097,4 +1097,14 @@ def foo
       end
     RUBY
   end
+
+  it 'does not register an offense for `if` without body' do
+    expect_no_offenses(<<~RUBY)
+      if some_condition
+        # skip
+      end
```

Comment:
That's just my two cents. Perhaps having no comments at all would better express being "without body" :-)

```suggestion
      if some_condition
      end
```